file-formats/aifd/type/zif_aff_aifd_v1.intf.abap

```diff
@@ -0,0 +1,40 @@
+INTERFACE zif_aff_aifd_v1
+  PUBLIC.
+  TYPES:
+    "! <p class="shorttext">AIF Interface</p>
+    "! AIF interface
+    BEGIN OF ty_aif_interface,
+      "! <p class="shorttext">Interface Object Name</p>
```

Comment:
Just a minor: 

Usually, we try to keep the title (= "shorttext") and the field name in sync. Would it make sense to decide either for "Object Name" or "Interface Object Name".

If I understood your explanation I guess I would go for just "Object Name" or even "Name" (the latter we used in various other object types)